wdl-lint/tests/lints/within-import-whitespace/source.errors

```diff
@@ -1,10 +1,18 @@
 note[ImportSort]: imports are not sorted lexicographically
-  ┌─ tests/lints/within-import-whitespace/source.wdl:9:1
+  ┌─ tests/lints/within-import-whitespace/source.wdl:8:1
   │
-9 │ import  "bar"  # BAD (2 spaces)
-  │ ^^^^^^^^^^^^^
+8 │ import "foo" as foo  # OK
+  │ ^^^^^^
```

Comment:
This is now tagging the first import, rather than the first import that is out-of-order (in this case, that'd be the second one).

Reply:
@adthrasher yes and no. `import "foo"` **is** out of order, once we've sorted the entire block which is why it's being flagged now. I suppose it's a matter of perspective... But I think this is fine, and the code is probably going to be more readable as I have it written than an alternative which flags `"bar"` instead (IMO at least 🤷 )